PanRangeSlider/Common/RangeSlider.cs

```diff
@@ -31,6 +31,71 @@ public class RangeSlider : TemplatedView
 
     public event EventHandler? UpperDragCompleted;
 
+    public static BindableProperty ThumbCornerRadiusProperty =
+    BindableProperty.Create(nameof(ThumbCornerRadius), typeof(double), typeof(RangeSlider), 10.0, propertyChanged: OnLayoutPropertyChanged);
+
+public static BindableProperty LowerThumbCornerRadiusProperty =
+    BindableProperty.Create(nameof(LowerThumbCornerRadius), typeof(double), typeof(RangeSlider), -1.0, propertyChanged: OnLayoutPropertyChanged);
+
+public static BindableProperty UpperThumbCornerRadiusProperty =
+    BindableProperty.Create(nameof(UpperThumbCornerRadius), typeof(double), typeof(RangeSlider), -1.0, propertyChanged: OnLayoutPropertyChanged);
```

Comment:
1) please fix the formatting (indention).
2) put props to the appropriate place